docs/sources/static/configuration/integrations/mssql-config.md

```diff
@@ -96,6 +96,21 @@ Full reference of options:
     [- <queries> ... ]]
 ```
 
+### Authentication
+By default, the `USERNAME` and `PASSWORD` used within the `connection_string` should correspond with a SQL Server username and password.
+
+If the Grafana Agent is running in the same Windows domain as the SQL Server, then the parameter `authenticator=winsspi` can be used within the `connection_string` without any additional credentials in order to authenticate.
```

Comment:
```suggestion
If {{< param "PRODUCT_ROOT_NAME" >}} is running in the same Windows domain as the SQL Server, then you can use the parameter `authenticator=winsspi` within the `connection_string` without any additional credentials to authenticate.
```

Reply:
@clayton-cornell PRODUCT_ROOT_NAME does not exist for static docs.